front/src/modules/ui/board-card-field-inputs/components/BoardCardEditableFieldDate.tsx

```diff
@@ -0,0 +1,39 @@
+import { ChangeEvent, useState } from 'react';
+
+import { BoardCardEditableField } from '@/ui/board-card-field/components/BoardCardEditableField';
+import { InplaceInputDateDisplayMode } from '@/ui/inplace-inputs/components/InplaceInputDateDisplayMode';
+import { InplaceInputDateEditMode } from '@/ui/inplace-inputs/components/InplaceInputDateEditMode';
+import { InplaceInputTextDisplayMode } from '@/ui/inplace-inputs/components/InplaceInputTextDisplayMode';
+import { InplaceInputTextEditMode } from '@/ui/inplace-inputs/components/InplaceInputTextEditMode';
+
+import { BoardCardEditableFieldDateEditMode } from './BoardCardEditableFieldDateEditMode';
+
+type OwnProps = {
+  value: Date;
+  onChange: (newValue: Date) => void;
+  editModeHorizontalAlign?: 'left' | 'right';
+};
+
+export function BoardCardEditableFieldDate({
+  value,
+  onChange,
+  editModeHorizontalAlign,
+}: OwnProps) {
+  const [internalValue, setInternalValue] = useState(value);
```

Comment:
So `BoardCardEditableFieldDate` takes care of the date value state ad provides it to the generic `BoardCardEditableField`.

This pattern is slightly different from the one applied in `EditableCellDate` which does not define the value state but instead receives it from the level above (Column in this case).